### PR TITLE
Blockchain as a form of E-voting

### LINKED ISSUE #4254: [Blockchain] Blockchain as a form of e-voting
### Introduction

Satoshi Nakamoto, in his famous paper *Bitcoin: A Peer-to-Peer Electronic Cash System* begins by talking about how the Internet's commerce and transaction system suffers from a trust-based model. Furthermore, Nakamoto mentions this model's need for a central authority that acts as an intermediary for transactions, that is, any and all transactions must go through this central authority. Think, for example, of a bank. Nakamoto's proposal in his paper is precisely that of a decentralized and highly secure technology as an alternative to this current model: the Blockchain.  

Drawing a parallel with a voting process, its integrity and effectiveness also depend on voters' trust and in the authorities that organized it. Electronic voting can improve some aspects of manual voting such as the speed, simplicity, and flexibility of the process. On the other hand, they bring red flags that deserve attention and care, such as issues of secrecy, privacy, transparency, and security.  Some places around the world have used some form of electronic voting or experimented with this voting model. In my country, Brazil, electronic voting machines have been used for elections since the 1990s, for example. In Moscow, an e-voting experiment was recently carried out using Blockchain as a backend.

In the words of The Economist, the Blockchain is a way of *" creating and preserving truths"*. In fact, the technology has promising properties for an electronic voting system such as anonymity, decentralization, transparency, immutability, and encryption. In this article, we will briefly see what Blockchain is, then we will see how this technology can be used in electronic voting, its advantages and problems, and end by talking a little about Moscow's experience using Blockchain in an election.

### Key Takeaways

- Quick revision of Blockchain, how it works, and its basic properties;
- How can Blockchain be used in e-voting systems;
- Advantages and problems of using Blockchain in e-voting;
- Example: how was Moscow's e-voting experience using Blockchain.

### References

- [Blockchain for Electronic Voting System — Review and Open Research Challenges.](https://www.mdpi.com/1424-8220/21/17/5874)
- [The great chain of being sure about things.](https://www.economist.com/briefing/2015/10/31/the-great-chain-of-being-sure-about-things)
- [Bitcoin: A Peer-to-Peer Electronic Cash System.](https://bitcoin.org/bitcoin.pdf)
- [How Moscow organized voting on a Blockchain in 2020.](https://ict.moscow/en/news/how-moscow-organized-voting-on-blockchain-in-2020/)

### PR DESCRIPTION
**Pull request by csamuelsm**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Topic suggestion this closes

This closes #4254

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/blockchain-as-a-form-of-e-voting/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Blockchain as a form of e-voting
+
+### **Introduction**
+
```
Comment (Bayler) [inline content/articles/blockchain-as-a-form-of-e-voting/index.md:4]: No need to put asterics (**) on headings as they are bold by default. Please remove them in all your headings.

File: content/articles/blockchain-as-a-form-of-e-voting/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Satoshi Nakamoto, in his famous paper *Bitcoin: A Peer-to-Peer Electronic Cash System* [1] begins by talking about how the Internet's commerce and transaction system suffers from a trust-based model. Furthermore, Nakamoto mentions this model's need for a central authority that acts as an intermediary for transactions, that is, all the transactions must go through this central authority. Think, for example, of a bank. Nakamoto's proposal in his paper is that of a decentralized and secure technology as an alternative to this current model: the Blockchain.
```
[the remark this answers is not in the document]
Comment (Bayler) [reply content/articles/blockchain-as-a-form-of-e-voting/index.md:5]: Please note that the way you've referenced inside your article is the way you reference academic papers. Github doesn't recognize that. You should add references the same you've done in the `reference` section. I've shown you how that is done above. First, commit the suggestion above, and then check the whole of your article and change accordingly.

**Comment by csamuelsm** [comment]
Hey @Bayler! I committed the requested changes, except for the `Prerequisites` section. Could you take a look at it? Thanks.

File: content/articles/blockchain-as-a-form-of-e-voting/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bayler) [reply content/articles/blockchain-as-a-form-of-e-voting/index.md]: @csamuelsm I don't see any documentation stating that it should be there, but for the benefit of our readers, you could include one.

File: content/articles/blockchain-as-a-form-of-e-voting/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- Introduction
+- Blockchain
+    - Features of Blockchain
+    - Peer-to-Peer network
+    - Transaction
+    - The Chain and the Blocks
+    - Mining and Proof of Work
+- E-voting and Blockchain: Advantages and Disadvantages
+- A Real-Life Example: How Moscow Implemented a Blockchain Vote
+- Conclusion
```
Comment (Bayler) [inline content/articles/blockchain-as-a-form-of-e-voting/index.md:11]: Please note that these should be links. When a reader clicks on it, it should take the reader to the respective section. Check out this [article](https://github.com/section-engineering-education/engineering-education/pull/4148/files) for reference.

**Comment by hectorkambow** [comment]
@csamuelsm 
Good afternoon and thank you for submitting your topic to the EngEd program. It seems this PR is still quite a bit aways from being completed. 

As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
All revisions and edits should be completed on your own forked repo (as to not build up a backlog in the queue). 

@Bayler if the PR still needs more work after your tech review - then we will be closing this PR to allow the author the time needed to make the required changes. 

Then they can re-submit when it is ready to be published as a new PR (and we can add the duplicate label). 

NOTE: ANY articles that coming in that require this level (or a handful of edits) should be closed. 👍

**Comment by ahmadmardeni1** [comment]
Hey @csamuelsm 
Great article, it would be really great if you talk about DAOs in your article. It is a very important topic now and voting power is the key feature. I will leave some resources for you:
- [A beginner’s guide to DAOs - by Linda Xie](https://linda.mirror.xyz/Vh8K4leCGEO06_qSGx-vS5lvgUqhqkCz9ut81WwCP2o)
- [What is a DAO in Crypto?](https://www.youtube.com/watch?v=KHm0uUPqmVE)

NOTE: No need to be comprehensive, just a quick explanation.
Please @ me when you are done.
Cc @Bayler

**Comment by csamuelsm** [comment]
Hey @ahmadmardeni1 Added a quick explanation of DAOs. Thank you.

**Comment by ahmadmardeni1** [comment]
Great work gentlemen @csamuelsm @Bayler

**Pull request merged**